src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -78,7 +78,9 @@ export function getMdAutocompleteMissingPanelError(): Error {
     '[attr.aria-activedescendant]': 'activeOption?.id',
     '[attr.aria-expanded]': 'panelOpen.toString()',
     '[attr.aria-owns]': 'autocomplete?.id',
-    '(focus)': 'openPanel()',
+    // Note: we use `focusin`, as opposed to `focus`, in order to open the panel
+    // a little earlier. This avoids issues where IE delays the focusing the input.
```

Comment:
Typo: `IE delays the focusing the input` -> `IE delays the focusing of the input`